graylog2-web-interface/src/logic/notifications/NotificationsFactory.js

```diff
@@ -348,6 +374,21 @@ class NotificationsFactory {
             </span>
           ),
         };
+      }
+
+      case 'archiving_summary':
+        return {
+          title: 'Some archiving jobs did not complete successfully',
+          description: (
+            <span>
+              Recent failed attempts: {' '}
+              <ul>
+                {notification.details.archiveErrors.map((error) => <li key={error.id}>{error}</li>)}
```

Comment:
> The JS console is showing this when rendering the new notification: `Warning: Each child in a list should have a unique "key" prop` Can you suggest how to fix that?

When rendering arrays of elements, React needs to have a unique key to each element, so it can track when to re-render them. In this case, `error` is a string and does not have an `id` property, so it defaults to `undefined`, which is the same as not providing a value for it. Since in this case the array is coming from the backend and will not be changed, I think the easiest way is to use array indices for it:

```suggestion
                {/* eslint-disable-next-line react/no-array-index-key */}
                {notification.details.archiveErrors.map((error, idx) => <li key={idx}>{error}</li>)}
```

Using the index is not ideal and ESLint will complain about it, so I included the comment to disable it on that line. I still think that is better than using a relatively long sentence as key.